server/jetty/src/main/java/io/deephaven/server/jetty/JettyMain.java

```diff
@@ -2,28 +2,50 @@
 
 import io.deephaven.base.system.PrintStreamGlobals;
 import io.deephaven.configuration.Configuration;
+import io.deephaven.server.jetty.JettyConfig.Builder;
 import io.deephaven.server.runner.Main;
 
 import java.io.IOException;
+import java.time.Duration;
 import java.util.concurrent.TimeoutException;
 
 public class JettyMain extends Main {
     public static void main(String[] args)
             throws IOException, InterruptedException, ClassNotFoundException, TimeoutException {
         final Configuration config = init(args, Main.class);
 
-        // defaults to 5 minutes
-        int httpSessionExpireMs = config.getIntegerWithDefault("http.session.durationMs", 300000);
-        int httpPort = config.getIntegerWithDefault("http.port", 10000);
-        int schedulerPoolSize = config.getIntegerWithDefault("scheduler.poolSize", 4);
-        int maxInboundMessageSize = config.getIntegerWithDefault("grpc.maxInboundMessageSize", 100 * 1024 * 1024);
+        // Defaults defined in JettyConfig
+        int httpSessionExpireMs = config.getIntegerWithDefault("http.session.durationMs", -1);
+        String httpHost = config.getStringWithDefault("http.host", null);
+        int httpPort = config.getIntegerWithDefault("http.port", -1);
```

Comment:
I am leery of divorcing the default configs from either the .prop file or the actual config.getFoo() invocation, at least as a matter of changing conventions.

Reply:
I don't think there is much of a coherent convention in how we currently use config. At least, in a lot of cases we tell the "objects" that they are responsible themselves for decoded how to parse their own properties...

I'm hoping this PR will at least present some code that we can agree is "better than before", and I think the code can stand alone (but is a nod towards https://github.com/deephaven/deephaven-core/issues/2471). For example, nowhere do we document that you can configure the property `scheduler.poolSize`, nor that the default is 4.

I will concede though, due to this prompting, I think the code can be consolidated and documented better, which I will do... hoping to motivate more that it is "better than before".